Source/CarthageKit/BinaryProject.swift

```diff
@@ -1,16 +1,19 @@
 import Foundation
 import Result
 
+/// The binary project.
```

Comment:
Oh, no misunderstandings here: I have not fixed ***all*** warnings that SwiftLint shows. Especially those which indicate long files or long methods need to be refactored which I didn't want to do within this PR to keep this as non-intrusive as possible. So, I've just removed that documentation and there's also no need for a `TODO` here since there's the SwiftLint warning that will remind us to do something about the docs there. ;)